modules/administration/pages/prometheus.adoc

```diff
@@ -0,0 +1,214 @@
+
+In PromQL, an expression can evaluate to one of three types:
+
+* Instant vector: a set of time series containing a single sample for each time series, all sharing the same timestamp
+* Range vector: a set of time series containing a range of data points over time for each time series
+* Scalar: a numeric floating point value
+```At least, I'd remove it.  
+
+The core part of any PromQL query is the metric name, for example: ``http_requests_total``.
+Labels can be used as optional selectors.
+This example returns the total number of HTTP requests that have status ``200`` and method ``GET``:
+
+----
+http_requests_total{status="200", method="GET"}
+----
+
+For more information about PromQL, see the official Prometheus documentation: https://prometheus.io/docs/prometheus/latest/querying/basics/).
+
+
+
+== Exporters
+
+Exporters are libraries which help in exporting existing metrics from third-party systems as Prometheus metrics.
+Exporters are useful whenever it is not feasible to instrument a given application or system with Prometheus metrics directly.
+Multiple exporters can run on a monitored host to export local metrics.
+
+The Prometheus community provides a list of official exporters, and many others can be found as community contributions.
+For detailed information and an extensive list of exporters, see: https://prometheus.io/docs/instrumenting/exporters/.
+
+With {productname}{nbsp}4, you can set up the Server and Proxy to expose Prometheus metrics to provide insights about {productname} self-health.
+Metrics are available for these services:
+
+* Hardware and Operating System
+* Java Virtual Machines
+* Apache
+* Squid
+* PostgreSQL
+* {productname} internals
+
+The self-health metrics are made available by {productname} Java application combined with Prometheus standalone exporters, running as systemd daemons.
+
+{productname} requires these packages to be installed on the Server and the Proxy.
+The packages are shipped with {productname} Server and Proxy, but their respective systemd daemons are disabled by default.
+
+These exporter packages are shipped with {productname} Server:
+
+* Node exporter: [systemitem]``golang-github-prometheus-node_exporter``.
+See https://github.com/prometheus/node_exporter.
+* PostgreSQL exporter: [systemitem]``golang-github-wrouesnel-postgres_exporter``.
+See https://github.com/wrouesnel/postgres_exporter.
+* JMX exporter: [systemitem]``prometheus-jmx_exporter``.
+See https://github.com/prometheus/jmx_exporter.
+* Apache exporter: [systemitem]``golang-github-lusitaniae-apache_exporter``.
+See https://github.com/Lusitaniae/apache_exporter).
+
+These exporter packages are shipped with {productname} Proxy:
+
+* Node exporter: [systemitem]``golang-github-prometheus-node_exporter``.
+See https://github.com/prometheus/node_exporter.
+* Squid exporter: [systemitem]``golang-github-boynux-squid_exporter``.
+See https://github.com/boynux/squid-exporter.
+
+
+
+== Prometheus Configuration
```

Comment:
I would add a section preceding this one explaining how to setup Prometheus - similar to what we have below for Grafana, section "Procedure: Setting up Grafana"